test/commands/release.js

```diff
@@ -120,4 +120,28 @@ describe('container release', () => {
       .then(() => stdMocks.restore())
       .catch(() => stdMocks.restore())
   })
+
+  it('has release phase but no new release', () => {
+    let api = nock('https://api.heroku.com:443')
+      .get('/apps/testapp')
+      .reply(200, {name: 'testapp'})
+      .patch('/apps/testapp/formation', {
+        updates: [
+          {type: 'web', docker_image: 'image_id'}
+        ]
+      })
+      .reply(200, {})
```

Comment:
Is this the actual response when no changes are made? If so, can we use that instead of fetching the latest release at all?

Reply:
API always returns a 200, no matter whether changes were made or not.

Reply:
Ah, sorry. No, on a batch update, we always return the full list of process types for the app.